### ISSUE TITLE
Unexpected error on CLI on Mac OS X

### ISSUE THREAD

**Issue by verheyenkoen** [cypress-io/cypress#717]
I'm getting the following error when typing: `cypress open` in the command line for version 1.0.0.

```
An unexpected error occurred while verifying the Cypress executable.

Please search Cypress documentation for possible solutions:

https://on.cypress.io

Check if there is a GitHub issue describing this crash:

https://github.com/cypress-io/cypress/issues

Consider opening a new issue.
----------

TypeError: Cannot read property 'pipe' of null
----------

Platform: darwin (16.7.0)
Cypress Version: 1.0.0
```

Also, when running tests on the CLI, the output is not colored anymore.

**Comment by brian-mann** [comment]
Yeah I am seeing that too. Our linux + windows versions are passing in CI, so might just be OSX.

**Comment by bahmutov** [comment]
Can it be the child process spawn options? I bet it is child.stderr work we have done to dev null stream

**Comment by brian-mann** [comment]
Yeah it's definitely not a binary problem. Definitely with the CLI itself.

**Comment by brian-mann** [comment]
I am seeing the same error when installing 0.20.3 on the new CLI

CYPRESS_BINARY_VERSION=0.20.3 cypress install

**Comment by bahmutov** [comment]
Got to start mac ci too, Linux and windows are not catching this.

**Comment by brian-mann** [comment]
Yeah I did not manually test this either... I rolled back to `0.20.3` temporarily

**Comment by brian-mann** [comment]
This will require a new release of the CLI, hah.

**Comment by brian-mann** [comment]
The full stack trace in OSX...

![screen shot 2017-10-10 at 7 28 17 am](https://user-images.githubusercontent.com/1268976/31384424-f66ab368-ad8c-11e7-8d56-b63dc0f2f123.png)

**Comment by brian-mann** [comment]
Both `child.stdout` and `child.stderr` are null O_O

**Comment by brian-mann** [comment]
I see what the problem is. We assumed the options were always pipe... but when verify calls it uses `inherit` as `stdio`... how did we miss this

**Comment by brian-mann** [comment]
I'm just going to do a conditional here for windows vs everything else...

**Comment by tjcafferkey** [comment]
I get the following error, think it's related:

```
An unexpected error occurred while verifying the Cypress executable.

Please search Cypress documentation for possible solutions:

https://on.cypress.io

Check if there is a GitHub issue describing this crash:

https://github.com/cypress-io/cypress/issues

Consider opening a new issue.
----------

TypeError: Cannot read property 'pipe' of null
----------

Platform: win32 (10.0.10240)
Cypress Version: 1.0.0
```

**Comment by brian-mann** [comment]
Yes it's all the same issue. Going to be fixed once all tests pass for `1.0.1`

**Comment by brian-mann** [comment]
Fixed in [`1.0.1`](https://docs.cypress.io/guides/references/changelog.html#1-0-1).
